.github/workflows/benchmarks.yml

```
@@ -69,7 +69,7 @@ jobs:
       - uses: tlambert03/setup-qt-libs@v1
 
       - name: Setup asv
-        run: python -m pip install asv virtualenv
+        run: python -m pip install asv[virtualenv]
```

```suggestion
        run: python -m pip install "asv[virtualenv]"
```

? I'm not sure if it's needed but might be worthwhile?